components/gfx/paint_thread.rs

```diff
@@ -373,7 +375,7 @@ impl<C> PaintThread<C> where C: PaintListener + Send + 'static {
     pub fn create(id: PipelineId,
                   url: Url,
                   chrome_to_paint_chan: Sender<ChromeToPaintMsg>,
-                  layout_to_paint_port: Receiver<LayoutToPaintMsg>,
+                  layout_to_paint_port: IpcReceiver<LayoutToPaintMsg>,
```

Comment:
Ah, is this replacing a channel by an ipc channel? Is this the source of slowdown?